interface/reports/billing_tracker.php

```diff
@@ -0,0 +1,141 @@
+                ],
+                "order": [[1, 'asc']]
+            });
+
+            /* Formatting function for row details - modify as you need */
+            function format ( d ) {
+                // `d` is the original data object for the row
+                let claimsTable = '<table cellpadding="5" cellspacing="0" border="0" style="padding-left:50px;">'+
+                    '<tr>'+
+                    '<td>Jim Jones</td>'+
+                    '<td>BCBS NC</td>'+
+                    '<td>$150</td>'+
+                    '<td><a href="">details</a></td>'+
+                    '</tr>'+
+                    '<tr>'+
+                    '<td>Paula Smith</td>'+
+                    '<td>BCBS NC</td>'+
+                    '<td>$200</td>'+
+                    '<td><a href="">details</a></td>'+
+                    '</tr>'+
+                    '<tr>'+
+                    '<td>John Dow</td>'+
+                    '<td>BCBS NC</td>'+
+                    '<td>$50</td>'+
+                    '<td><a href="">details</a></td>'+
+                    '</tr>'+
+                    '</table>';
+
+                return claimsTable;
+            }
+
+            // Add event listener for opening and closing details
+            $('#billing-tracker-table tbody').on('click', 'td.details-control', function () {
+                var tr = $(this).parents('tr');
+                var row = oTable.row( tr );
+
+                if ( row.child.isShown() ) {
+                    // This row is already open - close it
+                    row.child.hide();
+                    tr.removeClass('shown');
+                }
+                else {
+                    // Open this row
+                    row.child( format(row.data()) ).show();
+                    tr.addClass('shown');
+                }
+            } );
+        });
+    </script>
+</head>
+<body>
+    <div id="container_div" class="mt-3">
+         <div class="w-100 p-4">
+
+             <table id="billing-tracker-table">
+                 <thead>
+                 <tr>
+                     <th>&nbsp;</th>
+                     <th><?php echo xl('Batch ID') ?></th>
+                     <th><?php echo xl('Status') ?></th>
+                     <th><?php echo xl('X-12 Partner') ?></th>
+                     <th><?php echo xl('File') ?></th>
+                     <th><?php echo xl('Date Created') ?></th>
+                     <th><?php echo xl('Date Updated') ?></th>
+                 </tr>
+                 </thead>
+                 <tfoot>
+                 <tr>
+                     <th>&nbsp;</th>
+                     <th><?php echo xl('Batch ID') ?></th>
+                     <th><?php echo xl('Status') ?></th>
+                     <th><?php echo xl('X-12 Partner') ?></th>
+                     <th><?php echo xl('File') ?></th>
+                     <th><?php echo xl('Date Created') ?></th>
+                     <th><?php echo xl('Date Updated') ?></th>
```

Comment:
xlt to above 6 lines

Reply:
Removed this file, was checked in by accident.